staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -44,11 +44,14 @@ type Webhook struct {
 
 	sourceFactory sourceFactory
 
-	hookSource       Source
-	clientManager    *webhookutil.ClientManager
-	namespaceMatcher *namespace.Matcher
-	objectMatcher    *object.Matcher
-	dispatcher       Dispatcher
+	hookSource                  Source
+	staticSource                Source
```

Comment:
I expected a single source that combined the API-based and file-based sources internally